wormhole-connect/src/views/v2/Bridge/AmountInput/index.tsx

```diff
@@ -199,7 +273,7 @@ const AmountInput = (props: Props) => {
         disabled={maxButtonDisabled}
         onClick={() => {
           if (props.tokenBalance) {
-            handleChange(sdkAmount.display(props.tokenBalance));
+            handleDebouncedChange(sdkAmount.display(props.tokenBalance));
```

Comment:
max button doesnt seem to work :(

https://github.com/user-attachments/assets/f2322fc5-c6fc-432b-8128-6ea15f14af9b